examples/llama_stack_client_examples/fireworks-compose.yaml

```diff
@@ -0,0 +1,16 @@
+services:
+  llamastack:
+    image: llamastack/distribution-fireworks
+    network_mode: "host"
+    volumes:
+      - ~/.llama:/root/.llama
+      - ./run.yaml:/root/llamastack-run-fireworks.yaml
```

Comment:
these mounts need exist locally; not sure where to retrieve `run.yaml`
`~/.llama` might be `~/.ollama`?